### DISCUSSION TITLE
MetaSpore使用体验

### DISCUSSION THREAD

**Discussion by Aqiang520** [meta-soul/MetaSpore#135] [General]
企业：企业
地区：中国北京
<img width="623" alt="截屏2022-06-07 下午2 39 13" src="https://user-images.githubusercontent.com/19472583/172312783-d1888a91-034c-4d19-a196-337cb52c2679.png">

使用场景：金融风控
联系方式:15321668208
感兴趣的功能点：分布式训练、在线预测、在线实验框架
建议或反馈：
1、平台提供的功能非常丰富，同时也提供了Jupyter功能，能够让算法童鞋更便捷的去进行各种算法实验，模型评估。集群打通了LakeSoul平台，能够支持spark任务。未来希望能够做成界面化的形式，比如Define the Model、Train the Model等部分，用户只需要准备好训练数据，根据业务场景选择合适的算法组件，所有的建模工作，包括数据读取、数据预处理、特征工程、模型训练、模型评估和模型发布，都可以通过托拉拽的方式去实现。目前刚接触，了解还不全面。后续会继续尝试使用平台。
2、AlphaIDE点击浏览器返回按钮，出现如下界面
<img width="632" alt="截屏2022-06-07 下午2 28 04" src="https://user-images.githubusercontent.com/19472583/172313107-e6a58f1b-fdd0-472f-b8c6-1f4d1ec0de81.png">

**Comment by intelligencegear** [comment]
感谢的您对MetaSpore产品所提的建议，您所提供的AlphaIDE中使用问题，我们会尽快修复。
